components/file_attachment/file_attachment.test.tsx

```diff
@@ -16,115 +16,162 @@ jest.mock('utils/utils.jsx', () => {
     };
 });
 
-function createComponent({fileInfo, handleImageClick, index, compactDisplay, canDownloadFiles = true, enableSVGs = false} = {}) { //eslint-disable-line react/prop-types
-    const fileInfoProp = fileInfo || {
-        id: 1,
+describe('FileAttachment', () => {
+    const baseFileInfo = {
+        id: 'thumbnail_id',
         extension: 'pdf',
         name: 'test.pdf',
         size: 100,
+        width: 100,
+        height: 80,
+        has_preview_image: true,
+        user_id: '',
+        create_at: 0,
+        update_at: 0,
+        delete_at: 0,
+        mime_type: '',
+        clientId: ''
+    };
+
+    const baseProps = {
+        fileInfo: baseFileInfo,
+        handleImageClick: jest.fn(),
+        index: 3,
+        canDownloadFiles: true,
+        enableSVGs: false
     };
-    const indexProp = index || 3;
-    const handleImageClickProp = handleImageClick || jest.fn();
-    return (
-        <FileAttachment
-            fileInfo={fileInfoProp}
-            handleImageClick={handleImageClickProp}
-            index={indexProp}
-            compactDisplay={compactDisplay}
-            canDownloadFiles={canDownloadFiles}
-            enableSVGs={enableSVGs}
-        />
-    );
-}
 
-describe('FileAttachment', () => {
     test('should match snapshot, regular file', () => {
-        const wrapper = shallow(createComponent());
+        const wrapper = shallow(<FileAttachment {...baseProps}/>);
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should match snapshot, regular image', () => {
         const fileInfo = {
-            id: 1,
+            id: 'thumbnail_id',
             extension: 'png',
             name: 'test.png',
             width: 600,
             height: 400,
             size: 100,
+            has_preview_image: true,
+            user_id: '',
+            create_at: 0,
+            update_at: 0,
+            delete_at: 0,
+            mime_type: '',
+            clientId: ''
```

Comment:
If there are no changes to these values across the tests, then I suggest sharing them with the `baseFileInfo`'s values. Each test's meaningful values would be much easier to scan.

```ts
const fileInfo = {
    ...baseFileInfo,
    extension: 'png',
    name: 'test.png',
    width: 600,
    height: 400,
    size: 100,
};
```